manifests/plugin/remote_execution/script.pp

```diff
@@ -49,6 +55,9 @@
   Enum['ssh', 'ssh-async', 'pull-mqtt'] $mode = 'ssh',
   Optional[Foreman_proxy::Sshloglevel] $ssh_log_level = undef,
   Boolean $cockpit_integration = true,
+  Optional[Integer] $mqtt_ttl = undef,
+  Optional[Integer] $mqtt_rate_limit = undef,
+  Optional[Integer] $mqtt_resend_interval = undef,
```

Comment:
To avoid misconfiguration we try to add upper and lower bounds when they make sense. So it's possible to use `Integer[LOWER]` and `Integer[LOWER, UPPER]` (so `Integer[1]` to accept 1 or higher). I'm not sure what the values should be, but I'm guessing a TTL should be at least 0 or 1 for example. Mind looking into that?

Reply:
I'm aware of the option to set bounds on types. However, the plugin itself does not do any kind of validation and having the installer impose additional layer of footgun prevention felt a bit weird